src/pages/waiting-room-to-car/cat-adi-part2/waiting-room-to-car.cat-adi-part2.page.html

```diff
@@ -32,9 +32,6 @@
 
             <transmission [formGroup]="form" [transmission]="pageState.transmission$ | async" (transmissionChange)="transmissionChanged($event)"></transmission>
 
-            <instructor-registration [formGroup]="form" [instructorRegistration]="pageState.instructorRegistrationNumber$ | async"
```

Comment:
You need to remove the mock of this component from the test file

Reply:
yeah, fair enough